lib/pos_consensus.go

```diff
@@ -665,12 +702,27 @@ func (cc *FastHotStuffConsensus) fetchValidatorListsForSafeBlocks(blocks []*MsgD
 		return nil, errors.Errorf("Error fetching next epoch entry after committed tip: %v", err)
 	}
 
+	// Fetch the PoS cutover validator
+	posCutoverValidator, err := BuildProofOfStakeCutoverValidator()
+	if err != nil {
+		return nil, errors.Errorf("Error building Proof of Stake cutover validator: %v", err)
+	}
```

Comment:
should we just move this into the if state on line 718? I guess it's not a particularly expensive operation so doesn't really matter much.

Reply:
Makes sense. It is safer to move to line 718